Singular/dyn_modules/Order/nforder_ideal.cc

```diff
@@ -310,6 +310,7 @@ nforder_ideal* nf_idPower(nforder_ideal* A, int i)
     return new nforder_ideal(A, 1);
   } else if (i<0) {
     Werror("not done yet");
+    return NULL;
```

Comment:
Looking at `Werror`, it seems it actually *can* return?!? If so, this return is needed. If I am mistaken, perhaps a `noreturn` annotation could be added to it and this `return` be removed again.

Reply:
Werror always returns: usually it write the error message and sets a flag (inerror) which is used by the interpreter:
the interpreter skips the curent result, returns to the top level and resets the flag.

Reply:
OK in that case I think my change is correct.